README.md

```diff
@@ -26,41 +26,43 @@
 This Karma plugin helps testing your UI5 projects.
 
 **Note:** This project has been renamed from `karma-openui5` to `karma-ui5` with the v1.0.0 release.  
-Checkout the [Migration Guide](./docs/migrate-v1.md) to see how to upgrade.  
-The `karma-openui5` documentation can be found on the [0.x branch](https://github.com/SAP/karma-ui5/tree/0.x#readme).
+For upgrade information, see the [Migration Guide](./docs/migrate-v1.md).  
+For the `karma-openui5` documentation, see [0.x branch](https://github.com/SAP/karma-ui5/tree/0.x#readme).
 
 
 ## Quickstart
 
 ### Installation
 
-First you need to install the Karma CLI globally
+First you need to install the Karma CLI globally:
 
 ```shell
 npm install -g karma-cli
 ```
 
 You can find more information on installing Karma [here](https://karma-runner.github.io/latest/intro/installation.html).
 
-Next you need to add `karma` and `karma-ui5` as devDependencies:
+Next, you need to add `karma` and `karma-ui5` as devDependencies:
 
 ```shell
 npm install --save-dev karma karma-ui5
 ```
 
-To start a browser you also need to install a launcher, e.g. for Chrome
+To start a browser, you also need to install a launcher, e.g. for Chrome:
+
 ```shell
 npm install --save-dev karma-chrome-launcher
 ```
 
 ### Configuration
 
-To configure this plugin you need to add two things to your `karma.conf.js`:
-1. Specify `"ui5"` in the list of `frameworks`
-1. Set a URL for serving the UI5 resources
-   - **Note:** This can be omitted when [using the UI5 Tooling](#url)
+To configure the plugin, you need to add two things to your `karma.conf.js`:
+1. Specify `"ui5"` in the list of `frameworks`.
+1. Set a URL for serving the UI5 resources.
+   - **Note:** This can be omitted when you use [UI5 Tooling](#url).
```

Comment:
> "you use UI5 Tooling"

Should it be "you use **the** UI5 Tooling" ?

Reply:
That would at least fit to the `url` option, isn't it?

> "When omitted and the project contains a `ui5.yaml` file, **the** [UI5 Tooling](https://github.com/SAP/ui5-tooling) will be used as server middleware."

Reply:
"you use UI5 Tooling" - No, because it is handled like a proper name. If it referred to a function in UI5 Tooling, you'd need the article: "you use the UI5 Tooling function." 

In the second case it was most likely my fault, I didn't see that URL was just the link and not actual text. Sorry for the confusion. So you have to remove the article in the second case.